index.js

```diff
@@ -889,6 +890,24 @@ module.exports = {
 				})),
 			);
 		},
+		marketsBidOn({ max = Infinity, account = undefined } = {}) {
+			return pageResults({
+				api: graphAPIEndpoints.binaryOptions,
+				max,
+				query: {
+					entity: 'optionTransactions',
+					selection: {
+						orderBy: 'timestamp',
+						orderDirection: 'desc',
+						where: {
+							type: 'bid',
+							account: account ? `\\"${account}\\"` : undefined,
+						},
+					},
+					properties: ['market'],
+				},
+			}).then(results => uniq(results.map(({ market }) => market)));
```

Comment:
I feel like bringing all of lodash into the webpack bundle is massive overkill for this - instead you can `reduce` it down to an object with `market` keys and then return `Object.values()`